dashboards/heroku-entrypoint.sh

```
@@ -4,6 +4,16 @@ set -e
+# live database instead of a snapshot of the credentials.
+export MB_DB_TYPE=postgres
+export MB_DB_CONNECTION_URI="${DATABASE_URL}?sslmode=require"
+
+# Bound the heap to the dyno's quota. Left unset, the JVM sizes the heap from
+# the host's memory rather than the dyno limit and Heroku reports R14.
+JAVA_HEAP="${JAVA_HEAP:--Xmx1900m}"
+
 # Run Metabase jar directly
 cd /app
-exec java -jar metabase.jar
+exec java $JAVA_HEAP -XX:+UseG1GC -jar metabase.jar
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)